test/rendering/cases/image-static-svg-bigwidth-bigheight/main.js

```diff
@@ -0,0 +1,33 @@
+import ImageLayer from '../../../../src/ol/layer/Image.js';
+import ImageStatic from '../../../../src/ol/source/ImageStatic.js';
+import Map from '../../../../src/ol/Map.js';
+import Projection from '../../../../src/ol/proj/Projection.js';
+import View from '../../../../src/ol/View.js';
+
+const extent = [0, -33614, 52354, 0];
```

Comment:
No, and it wouldn't have worked before #14444 either. The solution would be to handle extent aspect ratio different from image pixel ratio with a transform instead of replacing the loaded image with a canvas. That's beyond the scope of this pull request though.